test/integration/targets/win_optional_feature/aliases

```diff
@@ -0,0 +1,7 @@
+shippable/windows/group2
+skip/windows/2008
+skip/windows/2008-R2
+skip/windows/2012
+skip/windows/2012-R2
+skip/windows/2016
```

Comment:
My 2016 box comes with the DISM cmdlets, I would have thought we could run CI on some of these hosts. Probably use a more lightweight feature than .NET though.